src/tools/dev/xml/GenerateCMake.h

```diff
@@ -256,14 +256,36 @@ class CMakeGeneratorPlugin : public Plugin
     }
 
     void
-    FilterVTKLibs(std::vector<QString> &libs)
+    FilterVTKLibs(std::vector<QString> &libs,
```

Comment:
modification comment for all changes in this file?

Reply:
done.